### ISSUE TITLE
Rule: Conditionally executed code should be denoted by either indentation or curly braces

### ISSUE THREAD

**Issue by fmallet** [SonarSource/SonarJS#636]
Implements [RSPEC-3973](https://jira.sonarsource.com/browse/RSPEC-3973)

**Comment by nenadskobic** [comment]
This rule marks else statements incorrectly, ie. if we have:

```
if (something)
  doSomething();
else
  doSomethingElse();
```

`doSomethingElse()` line will be marked as incorrectly indented, which is nonsense.

Tried on SQ Comunity Version 8.3.1 (build 34397)

**Comment by vilchik-elena** [comment]
@nenadskobic indeed, could you create new issue for this False positive?
